packages/client/README.md

```diff
@@ -22,28 +23,80 @@ npm install @ucanto/client
 ```
 
 ## Example Usage
-```ts
-import * as Client from '@ucanto/client';
-import { ed25519 } from '@ucanto/principal';
+```js
+import * as Client from '@ucanto/client'
+import * as HTTP from '@ucanto/transport/http'
+import { CAR } from '@ucanto/transport'
+import { ed25519 } from '@ucanto/principal'
+import { Receipt, Message } from '@ucanto/core'
 
-const service = ed25519.Verifier.parse(process.env.SERVICE_ID);
-const issuer = ed25519.Signer.parse(process.env.CLIENT_KEYPAIR);
+const service = ed25519.parse(process.env.SERVICE_ID)
+const issuer = ed25519.parse(process.env.CLIENT_KEYPAIR)
 
-const invocation = await Client.invoke({
+// Mock fetch that simulates a UCAN service
+const mockFetch = async (url, input) => {
```

Comment:
I'm not sure if it's a good idea for a mock to be in the README, perhaps in a runnable code _example_ in the repo. IMHO the README should shown how to use the library against an already running server - i.e. just the code you'd need to use the client. I'd link to the _server_ README for actually setting up and running a server component.